src/cpp/src/timer.hpp

```diff
@@ -31,6 +31,6 @@ class ManualTimer {
     }
 
     ~ManualTimer() {
-        std::cout << m_title << ": " << m_total / 1000. << " secs" << std::endl;
+        // std::cout << m_title << ": " << m_total / 1000. << " secs" << std::endl;
```

Comment:
Remove?

Reply:
Decided to comment for debug purposes, best decision is move such prints under `Debug` flag